android/java/org/chromium/chrome/browser/settings/BraveAccountPreference.java

```diff
@@ -0,0 +1,59 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.settings;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.util.AttributeSet;
+import android.widget.TextView;
+
+import androidx.preference.PreferenceViewHolder;
+
+import org.chromium.build.annotations.NullMarked;
+import org.chromium.build.annotations.Nullable;
+import org.chromium.chrome.R;
+import org.chromium.components.browser_ui.settings.ChromeBasePreference;
+
+/** A preference for Brave Account that supports customizing title and summary text colors. */
+@NullMarked
+public class BraveAccountPreference extends ChromeBasePreference {
+    private final int mTitleTextColor;
+    private final int mSummaryTextColor;
+
+    public BraveAccountPreference(Context context, @Nullable AttributeSet attrs) {
+        super(context, attrs);
+
+        TypedArray a = context.obtainStyledAttributes(attrs, R.styleable.brave_account_preference);
+        mTitleTextColor = a.getColor(R.styleable.brave_account_preference_title_text_color, 0);
+        mSummaryTextColor = a.getColor(R.styleable.brave_account_preference_summary_text_color, 0);
```

Comment:
does that work correct when you are on the pref screen and change theme from dark to light or light to dark? The preference usually isn't recreated and we can have stale colors in that case,

Reply:
I double-checked, and the colors do update correctly on theme changes, even though they're not resolved at the ideal time. I'll still resolve them at bind time, since that's the safer approach. 👍🏻

Reply:
https://github.com/brave/brave-core/pull/32319/commits/77995e85e01fb8843e1807bac26f36a073660ae3